pkg/client/client.go

```diff
@@ -205,6 +212,13 @@ func (cli *Client) SetSecure() error {
 	return nil
 }
 
+// UseCookies indicates session based authentication approach and cookies are used to avoid creating new sessions
+// https://www.cisco.com/c/en/us/td/docs/switches/datacenter/nexus9000/sw/6-x/programmability/guide/b_Cisco_Nexus_9000_Series_NX-OS_Programmability_Guide/b_Cisco_Nexus_9000_Series_NX-OS_Programmability_Guide_chapter_011.html
```

Comment:
@leslie-qiwa , this is too long. Just say `UseCookies indicates session based authentication`

Reply:
done. removed.